nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PutDatabaseRecord.java

```diff
@@ -270,6 +274,7 @@ public class PutDatabaseRecord extends AbstractProcessor {
             .build();
 
     static final PropertyDescriptor TRANSLATE_FIELD_NAMES = new Builder()
+            .required(true)
             .name("put-db-record-translate-field-names")
             .displayName("Translate Field Names")
```

Comment:
I feel it is valuable to make these property names as clear as possible because this is a sophisticated feature.
I think another word besides "Translate" could be found to communicate what this feature does.
Here are some property display names I have thought of. I think Normalize is clearer. What to people think?
"Translate Field and Column Names for Comparison"
"Normalize Field and Column Names for Comparison"
"Adjust Field and Column Names for Comparison"
"Filter Field and Column Names for Comparison"

Reply:
Hi @exceptionfactory and @exceptionfactory 
if you have any thought or suggestion regarding @jrsteinebrey above comment please let me know